### ISSUE TITLE
Add support for Kong 0.13 routes and services

### ISSUE THREAD

**Issue by mlehner616** [kevholditch/terraform-provider-kong#22]
Are there any plans to add support for kong 0.13 /routes and /services?

**Comment by kevholditch** [comment]
Yes it will be added when I get time (I haven’t got much over the next few weeks). If you wish to contribute this I’d be happy to help with any questions. The support is there in [gokong](https://github.com/kevholditch/gokong) which is the API library used to talk to kong so shouldn’t be hard to add...
